pages/dev/governance/community-pool-proposals.mdx

```diff
@@ -51,8 +56,12 @@ $ axelard tx gov submit-proposal community-pool-spend <file> --from validator --
 		"recipient": "<fund_recipient_address>",
 		"deposit": "2000000000uaxl"
 }
+
 ```
 
+Once the proposal passes or fails (without `NoWithVeto`), the deposit will be refunded.
```

Comment:
Add the command/docs on how someone else can provide 2000 AXL. Feel free to test it out on testnet/devnet

Reply:
Could you help me with the full command so that I can update it as requested?